SR2020/2019-10-21-kit-distribution-and-kickstart-details.md

```diff
@@ -0,0 +1,22 @@
+---
+to: sr2020-teams
+subject: Kit Distribution & Kickstart details
+attachments:
+  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
+---
+
+Hi all,
+
+Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details of shipping your kits.
```

Comment:
```suggestion
Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details for shipping your kits.
```

Reply:
Actually on second thought, move the last sentence to the end of the next paragraph. (Makes it clearer that we want normal attendees to sign the form too, otherwise I think it could be misread if being scanned)